Apps/WebClient/src/ClientApp/src/components/report/ReportsComponent.vue

```diff
@@ -131,18 +135,42 @@ export default class ReportsComponent extends Vue {
     }
 
     get headerData(): ReportHeader {
-        return {
-            phn: this.patient.personalHealthNumber,
-            dateOfBirth: this.formatDate(this.patient.birthdate || ""),
-            name: this.patient
-                ? this.patient.preferredName.givenName +
-                  " " +
-                  this.patient.preferredName.surname
-                : "",
-            isRedacted: this.reportFilter.hasMedicationsFilter(),
-            datePrinted: new DateWrapper(new DateWrapper().toISO()).format(),
-            filterText: this.reportFilter.filterText,
-        };
+        const dependent = this.dependents.find(
+            (d) => d.dependentInformation.hdid === this.hdid
+        );
+        if (dependent) {
+            return {
+                phn: dependent.dependentInformation.PHN,
+                dateOfBirth: this.formatDate(
+                    dependent.dependentInformation.dateOfBirth || ""
+                ),
+                name: dependent.dependentInformation
+                    ? dependent.dependentInformation.firstname +
+                      " " +
+                      dependent.dependentInformation.lastname
+                    : "",
+                isRedacted: this.reportFilter.hasMedicationsFilter(),
+                datePrinted: new DateWrapper(
+                    new DateWrapper().toISO()
+                ).format(),
+                filterText: this.reportFilter.filterText,
+            };
```

Comment:
To reduce repetition, I'd suggest refactoring this into a separate function that accepts a phn, dob, firstName, and lastName and returns the object with all of the extra properties and required formatting.